services/noirlab/interceptors/request.ts

```diff
@@ -0,0 +1,23 @@
+import { RequestOptions } from "@hey-api/client-next";
+import { env } from "@/env";
+
+const isServer = typeof window === "undefined";
+
+const addCaching = (options: RequestOptions) => {
+  const isMedia = new RegExp(/\/media\/(images|videos)\//).test(options.url);
+
+  if (isServer) {
+    if (!options.cache) {
```

Comment:
respect any cache already applied, otherwise explicitly force cache. This will be important for a Next 15 upgrade where caching is no longer the default.